README.md

```diff
@@ -106,13 +106,17 @@ You can run the tests with `cargo test`.
 
 ## Built With
 
-External crates :
+External crates:
 
-- `curl` for sending HTTP requests.
-- `serde_json` for JSON parsing.
-- `percent-encoding` to encode search queries in percent encoding.
-- `termion` and `tui` for the text user interface
+- `base64` 
 - `clap` for the command line interpreter
+- `failure`
```

Comment:
```suggestion
```

Reply:
Isn't that in the `Cargo.toml`?

https://github.com/lethenju/spotify_rust/blob/635850eb57ef81001400715a26383f521793dc46/Cargo.toml#L11

Reply:
What about that?

```suggestion
- `failure` for error handling
```